paper.md

```diff
@@ -0,0 +1,178 @@
+
+# HNN-core facilitates reproducibility and computationally expensive workflows
+
+The HNN GUI and its corresponding tutorials are beneficial for novice users to learn how to interact with the neocortical model to study the multiscale origin of source localized MEG/EEG signal. The interactive GUI allows users to quickly visualize how changes in parameters impact the simulated current dipole along with simultaneous changes in layer specific cell activity to test hypotheses on the mechanistic origins of recorded MEG/EEG waveforms. While the GUI is advantageous for learning how to study the multiscale origin of MEG/EEG sources, its functionality is limited as it only enables manipulation of a subset of GUI exposed parameters. Scripting in HNN-core greatly expands the software utility particularly for large research projects where reproducibility and batch processing is of key importance. The scripted interface allows multi-trial simulations enabling the use of computationally expensive parameter optimization algorithms and parameter sweeps using parallel processing on computer clusters. HNN scripting also facilitates the creation of publication-quality figures and advanced statistical analysis. Further, the software can be integrated with existing scripted workflows, such as those developed in MNE-Python [@gramfort2013meg], a well-established source localization software, enabling source localization and circuit interpretation in just a few lines of code (see tutorial in the HNN-core documentation).
```

Comment:
```suggestion
The HNN GUI and its corresponding tutorials are beneficial for novice users to learn how to interact with the neocortical model to study the multiscale origin of source localized MEG/EEG signals. The interactive GUI allows users to quickly visualize how changes in parameters impact the simulated current dipole along with simultaneous changes in layer specific cell activity to test hypotheses on the mechanistic origins of recorded MEG/EEG waveforms. While the GUI is advantageous for learning how to study the multiscale origin of MEG/EEG sources, its functionality is limited as it only enables manipulation of a subset of GUI exposed parameters. Scripting in HNN-core greatly expands the software utility particularly for large research projects where reproducibility and batch processing is of key importance. The scripted interface allows multi-trial simulations enabling the use of computationally expensive parameter optimization algorithms and parameter sweeps using parallel processing on computer clusters. HNN scripting also facilitates the creation of publication-quality figures and advanced statistical analysis. Further, the software can be integrated with existing scripted workflows, such as those developed in MNE-Python [@gramfort2013meg], a well-established source localization software, enabling source localization and circuit interpretation in just a few lines of code (see a tutorial of this in the HNN-core documentation).
```

- L93: "signal" should be plural
- L106: "a tutorial of this in the HNN-core documentation..."